### ISSUE TITLE
Normalize opacity on input placeholders

### ISSUE THREAD

**Issue by rienheuver** [argyleink/open-props#296]
Firefox apparently puts some reduced opacity on `:placeholder`. This means that the placeholder-color is different in firefox than other browsers. Open-props' normalize seems like the perfect place to set this straight with something along the lines of
```
:where(input:placeholder) {
  opacity: 1;
}
```

**Comment by argyleink** [comment]
nice observation, and yeah totally a normalize candidate 👍🏻 

i tested it really quick, in firefox and chrome, and i think it'd need to also include a text color:

```css
::placeholder {
  opacity: 1;
  color: var(--text-2);
}
```

But i feel like text-2 is too high contrast. I need to create a new prop that's a color right on the cusp of wcag contrast requirements, in both light and dark.

**Comment by rienheuver** [comment]
Completely agreed. I found this opacity-problem because I'm working on a set of components (buttons, inputs etc.) based on open-props (currently named open-components) where I also ran into the placeholder contrast problem. In my opinion it's actually fine with text-2 in dark mode but not in light. But maybe that's because my inputs have `background-color: var(--surface-3)`? 

Here is the relevant excerpts from my WIP:
```
@media (prefers-color-scheme: light) {
  :where(html) {
    --text-3: var(--gray-6);
  }
}
@media (prefers-color-scheme: dark) {
  :where(html) {
    --text-3: var(--gray-4);
  }
}

input {
  background-color: var(--surface-3);
  
  &::placeholder {
    font-weight: 500;
    color: var(--text-3);
    opacity: 1;
  }
}
```

No clue if these fit within wcag requirements though. If this could be resolved within open-props that'd be even better of course, since my solution now feels like I'm somewhat extending open-props instead of just applying it.
